packages/components/src/Enumeration/Enumeration.scss

```diff
@@ -23,14 +23,19 @@ $tc-enumeration-font-size: 1.4rem !default;
 				padding: 0 $padding-normal;
 
 				svg {
-					fill: $tc-enumeration-svg-color;
 					height: $tc-enumeration-font-size;
 					width: $tc-enumeration-font-size;
 				}
 			}
 		}
 	}
 
+	btn-link {
```

Comment:
this is not a valid selector except if you have writte <btn-link>

Reply:
And I don't understand why this has been moved. It seems that the icon buttons are all .btn.btn-link so basically you have the same result as before the change.
Or am I missing something ?

Reply:
You're right, it was wrong and useless.